package.json

```
@@ -1,5 +1,8 @@
 {
   "name": "idp-profile-ui",
+  "engines": {
+    "node": "=16.17.0"
```

What would you think of something like this instead, to allow non-breaking newer versions?
```suggestion
    "node": ">=16.0.0 <17.0.0"
```